docs/FAQ/gui-faq-en.md

```diff
@@ -20,147 +20,155 @@ Zonemaster
+in case they get results other than color-coded "green" for any
+test of their domain name(s).
 
 <a name="q4"></a>
 #### 4. Zonemaster returns "Error" or "Warning" for my domain. What does it mean?
-It depends on which test failed for your domain.
+It depends on which test failed for your domain. Each test are accompanied with messages
+describing the issues found.
+You can also get further insight about each test from the [Implemented Test Cases] document.
 
 <a name="q5"></a>
-#### 5. How can Zonemaster judge what is right and wrong?
-There is no final judgement of the health of a domain that can be bestowed by
-anyone. The people behind Zonemaster do not claim that the tool is correct in
-every aspect. Sometimes opinions differ. We have done our very best to create a
-default policy for found errors within
-this project. Hopefully this is a good compromise between what is an actual
-potentially dangerous error and what could be merely seen as a notice or warning.
-The added advantage of the tool is that one can add a policy file suited to
-one's necessity to a specified directory and ask the tool to use that policy
-file when running the tests.
-But as with all things as evolving as DNS the situation is most likely
-changing, what is a notice today could be an error tomorrow. If you really think
-we have made a mistake in our judgement please do not hesitate to drop us an email
-at [zonemaster-users@lists.iis.se](mailto:zonemaster-users@lists.iis.se) (moderated
-mailing list) with a link to your test and an explanation why you think it
-shows something that you consider incorrect.
+#### 5. How can Zonemaster distinguish between what is right and wrong?
+There is not a definitive judgement of the health of a domain that can be bestowed by anyone.
+The people behind Zonemaster do not claim that the tool is correct in every aspect.
+Sometimes opinions differ.
+We are doing our very best to create a default policy for errors found within this project,
+with information primarly based on industry standards such as [RFCs].
+
+Hopefully you will find in our tool a good compromise between what is an actual,
+potentially dangerous *error* and what could be merely perceived as a *notice* or *warning*.
+You can find more information on our vision of severity level in the [Severity Level Definitions] document.
+However, note that one advantage of the tool is that it is always possible for one to add a policy file 
+better suited to one's needs and ask the tool to use that policy file when running tests.
+Such configuration is accessible through the "Options" button.
+
+Finally, as with all things as evolving as the DNS, what is a *notice* today could be an *error* tomorrow.
+If you think we have made a mistake in our judgement please do not hesitate to send us
+an email at [zonemaster-users@lists.iis.se] (moderated mailing list) with a link to your test
+result and an explanation as to why you think it shows something that you consider incorrect.
```

Comment:
I think this is too wordy and that it should not start by pointing out the uncertainty. Suggestion:
```
The judgement of Zonemaster is primarily based on the DNS standards as defined in the [RFCs]. It
also bases the judgement on best practices for DNS. All Zonemaster tests are defined in
[test case specifications][Implemented Test Cases document] in which the references to the standard
document for that test case is found.

In the [Severity Level Definitions] document the difference between *notice*, *warning* and *error*
is found.

Sometimes there are different interpretations of the standards or opinion on what is best practice,
and the Zonemaster Project is always open to input. If you think we have made a mistake in our
judgement please do not hesitate to send us an email at [zonemaster-users@lists.iis.se] (moderated
mailing list) with a link to your test result and an explanation as to why you think it shows
something that you consider incorrect.
```

Reply:
Updated (+ rephrased a bit)